wsl/os/almalinux9/README.md

```diff
@@ -123,7 +126,10 @@ Your virtualhost should be accessible and ready to use.
 
 You will install your project under the `html` directory of your project, for example: `/var/www/example.localhost/html`.
 
-The virtualhost's document root is set to the `public` directory of the above location, for example `/var/www/example.localhost/html/public`.
+### NOTE
```

Comment:
Shouldn't we prepend these notes to the already existing **Note** section below this one?